lib/index.js

```diff
@@ -158,8 +160,11 @@ api.openCollections = brCallbackify(async names => {
   // open the collections
   logger.debug('opening collections', {collections: unopened});
   const collections = {};
+  const {writeConcern} = api.db.options;
   await Promise.all(unopened.map(async name => {
-    collections[name] = await api.db.collection(name);
+    // FIXME: this is to get around a bug in mongodb node driver 3.6.4
+    // where the writeConcern from db is not being passed to collection
```

Comment:
We try not to add FIXMEs these days and `this` isn't clear to people without the context of this PR.

```suggestion
    // Note: We only pass `{writeConcern}` here to get around a bug in mongodb
    // node driver 3.6.4 where `writeConcern` from `db` is not passed to collection
```